src/components/layouts/Layout.tsx

```diff
@@ -0,0 +1,118 @@
+import { setBreadcrumbs } from '../../redux/features/breadcrumbSlice';
+import { Layout, Menu } from 'antd';
+import { Footer } from 'antd/es/layout/layout';
+import { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import Breadcrumbs from './Breadcrumbs';
+import './activestyle.css';
+const { Header, Content, Sider } = Layout;
+interface PrvateRouteProps {
+  children: React.ReactNode;
+}
+// eslint-disable-next-line react/prop-types
+const AppLayout: React.FC<PrvateRouteProps> = ({ children }) => {
```

Comment:
Please remove the eslint comment and use component name like 
`
function AppLayout ({ children }: PrvateRouteProps) {
return ();
}
`